Directory.Build.props

```diff
@@ -23,6 +23,16 @@
   <PropertyGroup Condition="'$(Configuration)'=='Release'">
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('net5')) Or $(TargetFramework.StartsWith('net6')) Or $(TargetFramework.StartsWith('net7')) ">
+    <Net5OrAbove>true</Net5OrAbove>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Net5OrAbove)' == '' ">
```

Comment:
This `false` tag is never used. Will you please remove?

Reply:
Here I am not sure what is the correct way for making false values in MSBuild.
Probably you are right and this is unnecessary.
I’ll take a look with condition, perhaps it can be simplified to $(Net5OrAbovr) without comparing to true.

Reply:
Actually it must be defined with false, otherwise MSBuild doesn't understand boolean type.